requirements-hyperopt.txt

```diff
@@ -7,3 +7,4 @@ scikit-learn==1.1.2
 scikit-optimize==0.9.0
 filelock==3.8.0
 progressbar2==4.0.0
+-r requirements-freqai.txt
```

Comment:
Why should installing hyperopt install all freqAI requirements?
freqAI should only be loaded when we actually use it (basically when the config enables it)- so hyperopt of a normal strategy must be possible without these dependencies.

Reply:
@richardjozsa this was your addition, you said it had something to do with your docker image? I guess this is related to the other comment regarding docker.